### PR TITLE
[ARIA-LABEL]: add attribute to fa-icon

### PR DESCRIPTION
**Pull request by snewcomer**
https://www.w3.org/TR/wai-aria/states_and_properties#aria-label

### CODE REVIEW

**Comment by buschtoens** [comment]
Thank you for your contribution!

Tests are failing because of yarn complaining about an incompatible engine (Node 8). 
This would be fixed by `yarn config set ignore-engines`. 

Though I immensely appreciate it, I'd rather you upgrade Ember in a separate PR.  😉

**Comment by snewcomer** [comment]
@buschtoens Popped off the upgrade.  So do I need to add a yarn lockfile or is this something you can fix?

**Comment by buschtoens** [comment]
[`ember try:each`](https://travis-ci.org/martndemus/ember-font-awesome/builds/245952828#L521) fails because [it attempts to install dependencies](https://travis-ci.org/martndemus/ember-font-awesome/builds/245952828#L529) that are not officially marked as compatible with Node 8, even though they almost certainly are. 

The `.travis.yml` needs to be modified a bit. If you can wait a bit, I'll upgrade this add-on to the latest ember-cli version and fix Travis. 

Feel free to nag me, if I have not done this in the next 12 hours. 😉

**Comment by buschtoens** [comment]
Please pop off c8f979fa042d0f391229fb6e652e0ee7d183b415. Then we're good to go.

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "test": "ember try:each"
   },
   "engines": {
-    "node": ">= 0.12.0"
+    "node": ">= 4"
```
Comment (buschtoens) [inline package.json]: Please remove this commit.

File: addon/components/fa-icon.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ const FaIconComponent = Ember.Component.extend({
   ],
 
   attributeBindings: [
+    'ariaLabelAttribute:aria-label',
```
Comment (buschtoens) [inline addon/components/fa-icon.js]: Change to:

```js
'ariaLabel:aria-label'
```

File: addon/components/fa-icon.js [hunk 3]
```diff
@@ -102,6 +103,10 @@ const FaIconComponent = Ember.Component.extend({
   ariaHiddenAttribute: computed('ariaHidden', function() {
     let ariaHidden = get(this, 'ariaHidden');
     return ariaHidden !== false ? 'true' : undefined;
+  }),
+
+  ariaLabelAttribute: computed('ariaLabel', function() {
+    return get(this, 'ariaLabel');
   })
```
Comment (buschtoens) [inline addon/components/fa-icon.js]: Undo this insertion.

**Comment by buschtoens** [comment]
Thank you very much for submitting this PR and making the requested changes! ♥️

**Pull request merged**